src/OICodeGen.cpp

```diff
@@ -1247,6 +1353,41 @@ bool OICodeGen::isEmptyClassOrFunctionType(drgn_type *type,
           drgn_type_num_members(type) == 0);
 }
 
+/*
+ * Returns true iff the provided type
```

Comment:
Unfinished comment

Reply:
Oh yes I remember this one! It's what I was writing when we all got distracted by the wikipedia redesign 😆